assets/src/components/demo/Demo.tsx

```diff
@@ -99,6 +99,7 @@ class Demo extends React.Component<Props, State> {
           subtitle={subtitle}
           primaryColor={color}
           accountId={accountId}
+          baseUrl="http://localhost:4000"
```

Comment:
Lets make this app.papercups.io by default

Reply:
oops forgot to revert, good catch!